src/schema/v2/partner/partner.ts

```diff
@@ -367,6 +386,58 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
           })
         },
       },
+      partnerAlertsHitsConnection: {
+        description: "A connection of search criteria hits from a Partner.",
+        type: PartnerAlertsHitsConnection,
+        args: pageable({
+          page: {
+            type: GraphQLInt,
+          },
+          size: {
+            type: GraphQLInt,
+          },
+          artistID: {
+            type: GraphQLString,
+          },
+        }),
+        resolve: async ({ _id }, args, { partnerSearchCriteriaHitsLoader }) => {
+          if (!partnerSearchCriteriaHitsLoader) return null
+          const { page, size, offset } = convertConnectionArgsToGravityArgs(
+            args
+          )
+
+          type GravityArgs = {
+            page: number
+            size: number
+            total_count: boolean
+            artist_id?: string
```

Comment:
I dont think the API is accepting `artist_id` args at the time nor in the way we are persisting in the [schema](https://github.com/artsy/gravity/blob/2049ddd10a5ab8d964e1bec5644b8bc5737d463b/db/schema.rb#L670-L678)

Maybe we burn that bridge when we need to support additional args?

Reply:
ahh copy 🍝 ; fixing